### ISSUE TITLE
What's the difference between MDN Guides (MDN Learning Area) and Curriculum?

### ISSUE THREAD

**Issue by jlduchaney** [mdn/curriculum#48]
### Please describe your problem or observation

I'm someone who's just beginning my coding journey, and I was looking into [MDN Guides (MDN Learning Area)](https://developer.mozilla.org/en-US/docs/Learn). Now I see that [Curriculum](https://developer.mozilla.org/en-US/curriculum/) has just come out. 

Can you please summarize, at a glance, the relationship/distinction between the two? (i.e. On what occasion do you recommend using one vs. the other? What content/pages overlap vs. can only be found in one? Is one more comprehensive than the other, or did Curriculum just rearrange all of the preexisting content from the MDN Learning Area?) In other words, what tradeoffs does a learner miss out on by moving forward with one resource over the other? Thanks!

**Comment by glmvc** [comment]
I have to add that the difference was _not entirely clear_ to me at first, and I was expecting something quite different, even though I read the [About article](https://developer.mozilla.org/en-US/curriculum/about-curriculum/) first and have been familiar with MDN and learning and working with it for more than 2 years.

Therefore, I can imagine that the concept of the MDN Learning Area and Curriculum can be confusing to someone who doesn't know MDN at all or has recently discovered it.

Like I said, I was also _confused_ and _surprised_ at first, asking and thinking:
- "Why can't I just go through the Learning Area? Isn't it the same?"
- "What is the point of reading through the Curriculum and always being linked back to the Learning Area?"
- "The Learning Area is structured to follow article by article (step by step), and I like this approach more as a beginner than searching for the right modules by myself." (personal preference; maybe it's just my learning type)
- "When I work through the curriculum, I feel like I have to open a lot of links in new tabs, which I don't like, because it feels like a checklist, but I can't check the items I've learned, so I quickly lose track. This stresses me out as a complete beginner." (again, personal preference and perhaps just a matter of personal learning style)

Personally, I think I understand the purpose of the curriculum now, but I also think it's **not clear enough for everyone**, _especially_ new web developers.

**Possible reasons** for this issue that I can think of are:
- A _missing_ **key fact/explanation** on the [Curriculum Overview page](https://developer.mozilla.org/en-US/curriculum/) and/or a **better introduction** on the [About page](https://developer.mozilla.org/en-US/curriculum/).
- A _missing_ **note** in the Learning Area itself
- The linked resources are (still) _mainly_ MDN (Learning Area; `/learn`) articles and **not external resources**. (I think this will change in the future as appropriate resources are found and chosen)
- Learning Area articles that **do not teach the ideals** described in the curriculum module. (AFAIK, there are content issues in `mdn/content` regarding structuring, wording and relevance)

---

In **summary**, I _feel_ that the MDN Curriculum is a **kind of structured checklist for articles in the Learning Area**, at least _for now_.  
Don't get me wrong, this is just my first impression after reading the "Getting started" and "Core" module articles.

---

Overall, I hope my feedback is helpful and that this issue is the right place for it, or should such feedback be posted in the general MDN discussion forum? If so, feel free to move it there :)

**Comment by chrisdavidmills** [comment]
> Thanks, all!
> 
> After reading your responses and skimming through both resources, my impression is that Curriculum is like the syllabus and MDN Guides/MDN Learning Area is like the actual textbook itself. (My brain likes analogies.)

@jlduchaney YES - that is a really good analogy for this situation. I like it, and will probably use it elsewhere ;-)

> As a side note, this is my first-ever post on GitHub, so I don't know the proper etiquette. Let me know if there's any action I'm supposed to be taking on my end to keep the forum neat and tidy (e.g. mark an issue resolved). Cheers!

No further action needed right now; you are doing everything right. At the point where we think this issue is resolved, we can close it, but I want to read @glmvc's post first and make sure any other points that come from there are addressed.

Thanks for the useful conversation.

**Comment by chrisdavidmills** [comment]
@glmvc I think your have highlighted some reasonable concerns here. It seems to me that:

1. We could maybe add something to the curriculum to explain how to use it (e.g. keep it in a pinned tab and open other resources in other tabs, maybe keep resource lists in bookmarks, etc.)
2. We could add some more explanation to the learning area saying how it relates to/differs from the curriculum, and how to use the two together
3. We should add a feature to the curriculum that allows logged in users to check items they have covered so they know where they are up to, and maybe even add new custom resources they find, to their own personal view of it. This is something we've already discussed as a future feature.

**Comment by glmvc** [comment]
Thanks for checking my feedback, @chrisdavidmills.

1. That would certainly be an improvement, I think. I also like @jlduchaney's analogy. Maybe it should be added somewhere too :)
2. I agree + it would be very cool if the [`/Learn` overview article](https://developer.mozilla.org/en-US/docs/Learn) had an **article header** similar to the one on the [`/curriculum` overview page](https://developer.mozilla.org/en-US/curriculum/) (see my image below for an idea). I don't know if this is technically and design-wise possible (in a simple way), but it's _just an idea_. It would create a "connection" between the two parts on MDN, and I don't know if that's wanted in this case... What do you think? Is this idea too far-fetched?
3. This would be a great feature, IMHO! As far as I understand you, it would be some kind of **checklist** (which I addressed earlier) / personal notebook / personal reading and learning list that _I would like as a beginner_.
4. I think to remember that there were issues and discussions about this matter regarding that `/Learn` is a **bit outdated and not ideally structured**. I am sure the content team has more input on this. But take the [Environment setup module](https://developer.mozilla.org/en-US/curriculum/getting-started/environment-setup/) as an example. In [2.2 File systems](https://developer.mozilla.org/en-US/curriculum/getting-started/environment-setup/#2.2_file_systems), there is an item "File naming best practices for the web ..." that _suggests_ using hyphens _or_ underscores. The linked resource ["Dealing with files"](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files) _suggests_ "to separate words with hyphens, rather than underscores". This is probably **just a small detail**, but in my opinion, we should be consistent here. It is certainly not a bad idea to **compare the items in the curriculum with the content in `/Learn`** and assess whether all the information is available, correct, consistent and up-to date. This probably involves some work.

I believe that if all these points are implemented _accordingly_, there will be a significant **improvement** in terms of _understanding_ the difference between `/Learn` and `/curriculum` and _how to work_ with them. Also point 3 would improve working with the curriculum a lot, especially for beginners.

5. "Add more external resources" is one to add, which I think will _automatically_ resolve in the future, and I understand that it's not so easy to find appropriate ones ...

---

![article-header-learn](https://github.com/mdn/curriculum/assets/95487764/6d94ffcb-c71f-4984-8cf1-f9663e2fe4c0)

---

Please let me know if there is anything else I can help you with :)

**Comment by caugner** [comment]
See also: https://github.com/orgs/mdn/discussions/736
